src/modules/shared/UploadBucket.tsx

```diff
@@ -21,19 +22,32 @@ export const UploadBucket = ({
   removeFile: (a: number) => void;
   error?: string;
 }) => {
-  const onChangeFile = (e) => {
-    handleChange(e);
-  };
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+
+  const onChangeFile = (files: FileList) => {
+    const allowedTypes = ['image/png', 'image/jpeg', 'image/jpg'];
+    const selectedFiles = Array.from(files);
 
+    const invalidFile = selectedFiles.find(file => !allowedTypes.includes(file.type));
+    
+    if (invalidFile) {
+      setErrorMessage(`Invalid file type: ${invalidFile.name}`);
+    } else {
+      setErrorMessage(null);
+      handleChange(selectedFiles);
+    }
+  };
   return (
     <Box>
       <input
         style={{ display: 'none' }}
         type='file'
-        id={'id'}
+        accept='image/*'
```

Comment:
Lets create value from allowed file types

can we do this `allowedType.join(', ')` ?